test/wave2.test.js

```diff
@@ -1,91 +1,97 @@
-const axios = require('axios');                    // Import axios.
+const axios = require('axios'); // Import axios.
 const MockAdapter = require('axios-mock-adapter'); // This is kind of like VCR.
-const result = require('../src/result.js');        // Import result handling.
+const result = require('../src/result.js');
+ // Import result handling.
```

Comment:
This should be a line comment on the line above.  I think your formatter goofed this up.